Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -497,6 +514,10 @@ static void read_environment_variables()
         if (StringView { *env } == "_LOADER_BREAKPOINT=1"sv) {
             s_do_breakpoint_trap_before_entry = true;
         }
+
+        if (StringView { *env }.starts_with("LD_LIBRARY_PATH"sv)) {
+            s_ld_library_path = *env + 16u; // Point the path to just the part after the '='
+        }
```

Comment:
How about
```suggestion
	// put this all the way before the checks and use it for them, no sense in calling `strlen` on it over and over again
        StringView env_string { *env };
        
        constexpr auto library_path_string = "LD_LIBRARY_PATH="sv;
        if (env_string.starts_with(library_path_string)) {
            // replace this with a StringView, it's only used as a StringView anyway.
            s_ld_library_path = env_string.substring(library_path_string.length());
        }
```

Reply:
Cool! I didn't even know that was possible.